cookbooks/fb_syslog/attributes/default.rb

```diff
@@ -43,9 +43,14 @@
       'comment' => 'Log anything info level or higher. A lot ' +
                    'of things go into their own file.',
       'selector' => '*.info;mail,authpriv,cron,' +
-        'local0,local1,local2,local3,local5,local6,local7.none',
+        'local0,local1,local2,local3,local4,local5,local6,local7.none',
       'action' => "-#{syslog_file}",
     },
+    'authlog' => {
+      'comment' => 'Log all auth stuff',
+      'selector' => 'auth,authpriv.*',
+      'action' => '/var/log/auth.log',
+    },
```

Comment:
This is a Debian thing, so it should probably be gated to Debian/Ubuntu, CentOS and friends use /var/log/secure for this stuff.

Reply:
well it should probably just have the right name on RHEL and friends. Not having the auth/secure logs is a bummer. Will do a conditional.